survivalstan/utils.py

```diff
@@ -19,7 +19,7 @@ def _summarize_survival(df, time_col, event_col, evaluate_at=None):
     table = survival_table_from_events(df[time_col], df[event_col])
     table.reset_index(inplace=True)
     ## normalize survival as fraction of initial_n
-    table['initial_n'] = table.loc[table['event_at'] == 0.0,'at_risk'][0]
+    table['initial_n'] = max(table.at_risk)
```

Comment:
Yes - it felt a little verbose & .. while it's pretty safe to assume that cumulative survival should always be decreasing there may be scenarios where that assumption doesn't hold (in theory?) .. so using max value feels a little cleaner.